labs/TDIDN.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: TDIDN
+parent: Labs #(parent: Labs)
+---
+# Lab Name
+Telecom Decentralized Identities Network
+
+# Short Description
+TDIDN develops a decentralized identity and smart contract system for telecom. It focuses on DID-based authentication, privacy control, and efficient solutions for blacklist management, SLAs, and billing using CDRs.
+
+# Scope of Lab
+The TDIDN project aims to develop a comprehensive decentralized identity and smart contract ecosystem tailored for the telecom industry. It leverages Hyperledger technologies to provide a secure, transparent, and efficient solution to various telecom-related challenges. The following outlines the scope of the lab for the TDIDN project:
```

Comment:
What Hyperledger technologies this will leverage? 
- Is this being built on Indy or one of the other ledger technologies within Hyperledger?
- What parts of the current identity projects within Hyperledger are being used?

Reply:
1. Yes, we are using Hyperledger Indy as the base ledger for the project 
2. In addition to Indy, we are leveraging Hyperledger Aries JS and Hyperledger Anoncreds